packages/dev-utils/src/devApp/render.tsx

```diff
@@ -80,6 +80,9 @@ export type DevAppPageOptions = {
   children?: JSX.Element;
   title?: string;
   icon?: IconComponent;
+  sideBarItem?: JSX.Element;
```

Comment:
We'd typically capitalize this as `sidebarItem`

Reply:
Will fix.